openapi.yml

```diff
@@ -203,6 +203,37 @@ paths:
                 $ref: '#/components/schemas/RawFile'
   /files/{fileID}/validate:
     get:
+      tags: ['ACH Files']
+      summary: Validate file
+      description: Validates the existing file. You need only supply the unique File identifier that was returned upon creation.
+      operationId: checkFile
+      security:
+        - bearerAuth: []
+        - cookieAuth: []
+      parameters:
+        - name: X-Request-ID
+          in: header
+          description: Optional Request ID allows application developer to trace requests through the systems logs
+          example: rs4f9915
+          schema:
+            type: string
+        - name: fileID
+          in: path
+          description: File ID
+          required: true
+          schema:
+            type: string
+            example: 3f2d23ee214
+      responses:
+        '200':
+          description: File validated successfully without errors.
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+        '400':
+          description: Validation failed. Check response for errors
+    post:
```

Comment:
Does server route [here](https://github.com/moov-io/ach/blob/master/server/routing.go) need to be updated as well for POST request?